avocado/plugins/runners/exec_test.py

```diff
@@ -119,15 +123,19 @@ def _is_uri_a_file_on_cwd(uri):
         return False
 
     def _get_env(self, runnable):
-        env = dict(os.environ)
+        clear_env = runnable.config.get("runner.exectest.clear_env", None)
+        if clear_env in ["all", "system"]:
```

Comment:
Does anything validate the allowed values for `clear_env`?  If there's no error message, it could be pretty confusing if you made a typo in `system` or `all`.

Reply:
Right, I haven't thought about that, I will add validation, thanks